orchagent/intfsorch.cpp

```diff
@@ -181,6 +181,10 @@ void IntfsOrch::doTask(Consumer &consumer)
     while (it != consumer.m_toSync.end())
     {
         KeyOpFieldsValuesTuple t = it->second;
+        MacAddress mac;
+        int mac_attr_set;
+
+        mac_attr_set = 0;
```

Comment:
Can you set this in one line?

Reply:
I'm not sure in the loop that 'int mac_attr_set = 0' will be execute every time? If do, then can set this in one line.

Reply:
It is no problem to put these codes in one line.

Reply:
you could follow the other vairables' logics and check mac is empty or not; you don't need to have this variable